libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -1210,6 +1247,67 @@ export const buildTigerSpecificFunctions = (
             });
         });
     },
+
+    getWorkspaceDataFilters: async (
+        workspaceId: string,
+        objectId: string,
+    ): Promise<WorkspaceDataFiltersResult> => {
+        try {
+            return await authApiCall(async (sdk) => {
+                const result = await sdk.entities.getEntityWorkspaceDataFilters({
+                    workspaceId,
+                    objectId,
+                });
+                return result.data;
+            });
+        } catch (error: any) {
+            throw convertApiError(error);
+        }
+    },
+
+    setWorkspaceDataFilters: async (
+        workspaceId: string,
+        workspaceDataFilter: WorkspaceDataFilter,
+    ): Promise<WorkspaceDataFiltersResult> => {
+        return authApiCall(async (sdk) => {
+            const result = await sdk.entities.createEntityWorkspaceDataFilters({
+                workspaceId,
+                jsonApiWorkspaceDataFilterInDocument: workspaceDataFilter,
+            });
+            return result.data;
```

Comment:
Do we really need to return value for a `setXXX` function? 

There is also no return value for function [setWorkspaceDataFiltersLayout](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts#L1206C5-L1206C34), is there?